app/helpers/filter-by.js

```diff
@@ -0,0 +1 @@
+export { default, filterBy } from 'ember-functional-helpers/helpers/filter-by';
```

Comment:
Can we also export in https://github.com/DockYard/ember-functional-helpers/blob/remove-compose/addon/index.js please